snap/snapcraft.yaml

```diff
@@ -46,3 +46,11 @@ parts:
     build-packages:
       - git # The script needs Git.
     override-build: ./snap/get_version
```

Comment:
This steps depends on the semver step to be built, no? Also, does it have access to it in its path?

Reply:
> This steps depends on the semver step to be built, no?

Yes. I'm not sure how snapcraft determines the order of parts which have no dependencies, but apparently it did build it in the order we need, because the build succeeded every time I tried. Still definitely better to make the dependency explicit, so I added

```
    after:
      - semver
```

to the `version` part.

> Also, does it have access to it in its path?

Yes, apparently parts are built in an environment where the output `bin` directory of the other parts is already in the PATH.

Reply:
ack!